docs/source/system_reference_guide/oss_documentation/doi_and_licensing.ipynb

```diff
@@ -0,0 +1,99 @@
+{
```

Comment:
<p>Can we clarify which community for "already used within the community"</p><p> </p><p>Maybe add an asterisk to the bolded: <strong>If you are a MAAP user developing code, we recommend submitting your repository to the </strong><a href="https://zenodo.org/communities/maap?q=&amp;l=list&amp;p=1&amp;s=10&amp;sort=newest" target="_blank"><strong>MAAP Community on Zenodo</strong></a><strong>.</strong> to help distinguish it from a headed?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MAAP-Project/maap-documentation/pull/357/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
Oops I missed the very first part of that comment when I made some adjustments. I'd have to look back at documentation, but I think it was just generally the Earth Science and OSS communities🤔 I can go back and add something like that though once I confirm!